docs/content/en/configuration/default.md

```diff
@@ -166,6 +166,8 @@ server:
   proxy:
     enable: true
     header: CF-Connecting-IP
+    # Set this to true to disable TLS validation of backend proxies using https
```

Comment:
thanks for adding a bit of comment in the doc so that other people are aware of this feature.
Can you also add a brief description in this file chproxy/docs/content/en/configuration/proxy.md ?
Because this is what people see in the proxy part on chproxy.org => https://www.chproxy.org/configuration/proxy

Reply:
Done!